en/components/drop_down.md

```diff
@@ -224,6 +224,38 @@ If the sample is configured properly, a list  of countries should be displayed a
     </button>
 </div>
 
+### igxDropDownItemNavigation directive
+
+To enable keyboard navigation for the igxDropDown component, the igxDropDownItemNavigation directive should be applied. It's recommended that the directive is applied to the element that is triggering the opening and closing the of the drop down. That element will also have the focus, and when using the directive it will handle all triggered events. By default the focus will always stay on the element that igxDropDownItemNavigation is applied on, and this happens because the igxDropDown doesn't focus its items by default.
+The following sample demonstrates an input that on click opens and closes igxDropDown instance. Applying the igxDropDownItemNavigation on the input itself, will enable keyboard navigation, when using arrow up and arrow down, and in the same the input will not loose the focus during that process.
+
+```
+<igx-input-group #inputGroupProvince (click)="toggleDDProvince()">
+    <input igxInput #inputProvince type="text"
+        [igxDropDownItemNavigation]="dropdownProvince"/>
+    <igx-suffix><igx-icon [name]="arrowProvince"></igx-icon></igx-suffix>
+</igx-input-group>
+<igx-drop-down #dropdownProvince>
+    <igx-drop-down-item *ngFor="let p of provinceData">
+        {{ p }}
+    </igx-drop-down-item>
+</igx-drop-down>
+```
+
+If you want the drop down items to receive focus then you need to explicitly set `allowItemsFocus` property of the igxDropDown to `true`. In that case you can apply the igxDropDownItemNavigation on the drop down itself, however this can also happen on the input again. Applying the directive on the drop down means that you can omit the value, because it is the drop down itself.
```

Comment:
Applying the directive on the drop down means that you can omit the value, because it is the drop down itself
This is not clear enough. Should we change value with reference?

Reply:
target?